From 9aa1efde5818008f9110bc6a282809a7799c9d91 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Sun, 19 Aug 2018 09:57:04 +0200
Subject: [PATCH 019/293] xlat: add IN_MASK_CREATE to inotify_flags.in

* xlat/inotify_flags.in (IN_MASK_CREATE): New constant, introduced by
Linux commit v4.19-rc1~115^2.
* tests/inotify.c (main): Update expected output.
---
 xlat/inotify_flags.in | 1 +
 1 files changed, 1 insertions(+)

diff --git a/xlat/inotify_flags.in b/xlat/inotify_flags.in
index db1d839..671bd0e 100644
--- a/xlat/inotify_flags.in
+++ b/xlat/inotify_flags.in
@@ -16,6 +16,7 @@ IN_IGNORED 0x00008000
 IN_ONLYDIR 0x01000000
 IN_DONT_FOLLOW 0x02000000
 IN_EXCL_UNLINK 0x04000000
+IN_MASK_CREATE 0x10000000
 IN_MASK_ADD 0x20000000
 IN_ISDIR 0x40000000
 IN_ONESHOT 0x80000000
-- 
1.7.12.4

